tools/tune/eq/mls_freq_resp.m

```diff
@@ -200,7 +200,7 @@
 legend(labels);
 
 if selftest
-        idx = find(f < f_hi);
+        idx = f < f_hi;
```

Comment:
I personally like more find to not let this confuse idx to be a boolean. What's the benefit, I don't think this caused issues?

Reply:
Line Number 204, `f(idx)`  is an `array` or `cell array' . The performance is improved using logical indexing instead of FIND. If we are not considering boolean or logical array indexing , then shape of  the  output vector f(idx) could change . To retain the same output shape, we should replace find(f(idx)) by f(:).